cablelabs/transparent-security
# Telem rpt 2.0 update

#### What does this PR do?
Fixes #140 
Fixes #151 (duplicate)

Changes to the telemetry report to reflect the final 2.0 specs (https://github.com/p4lang/p4-applications/blob/master/telemetry/specs/telemetry_report.mdk)

#### Do you have any concerns with this PR?
No

#### How can the reviewer verify this PR?
Run automated tests, check Jenkins CI status, and/or view the attached PCAP file using int.lua
[core-eth3.zip](https://github.com/cablelabs/transparent-security/files/4548998/core-eth3.zip)

#### Any background context you want to provide?
The spec changed since implementing.
#### Screenshots or logs (if appropriate)
#### Questions:
- Have you connected this PR to the issue it resolves? Fixes #151
- Does the documentation need an update?
Will need to update the docs in PR: https://github.com/cablelabs/transparent-security/pull/131
- Does this add new dependencies? no
- Have you added unit or functional tests for this PR? no
